tanner/config.py

```diff
@@ -21,6 +21,8 @@
                                         'php_code_injection': True, "crlf": True},
                    'SQLI': {'type': 'SQLITE', 'db_name': 'tanner_db', 'host': 'localhost', 'user': 'root',
                             'password': 'user_pass'},
+                   'MySQLI': {'type': 'MySQL', 'db_name': 'tanner_db', 'host': '127.0.0.1', 'user': 'root',
```

Comment:
Don't change the code just to make tests. You should change the values in this `'SQLI': {'type': 'SQLITE', 'db_name': 'tanner_db', 'host': 'localhost', 'user': 'root',
                            'password': 'user_pass'}` for tests.

Reply:
I thought it will be useful. No worries will keep that in mind :)